examples/prom-statsd-exporter/conf.yaml

```diff
@@ -67,6 +67,13 @@ mappings: # Requires statsd exporter >= v0.6.0 since it uses the "drop" action.
     labels:
       err_type: "$1"
 
+  - match: "ratelimit_server.total_requests"
```

Comment:
I know we currently only have 1 API/handler, but it might make sense for these metrics to include the name of the API in it so that it can be easily extended in the future

Reply:
refactored to include name of gRPC method into the metrics